cluster/addons/fluentd-elasticsearch/README.md

```diff
@@ -72,7 +72,7 @@ avoid Fluentd pods scheduling there.
 [xPack]: https://www.elastic.co/products/x-pack
 [setupCreds]: https://www.elastic.co/guide/en/x-pack/current/setting-up-authentication.html#reset-built-in-user-passwords
 [fluentdCreds]: https://github.com/uken/fluent-plugin-elasticsearch#user-password-path-scheme-ssl_verify
-[fluentdEnvVar]: https://docs.fluentd.org/v0.12/articles/faq#how-can-i-use-environment-variables-to-configure-parameters-dynamically
+[fluentdEnvVar]: https://docs.fluentd.org/v/0.12/articles

```

Comment:
this is not a useful link for referring to fluentd envvars

Reply:
https://docs.fluentd.org/v0.12/articles/faq#how-can-i-use-environment-variables-to-configure-parameters-dynamically is gone!Maybe we need not to update it! i update the pr.

Reply:
do you have any question?

Reply:
if the article is gone, just remove the link and reference